imports/ui/StudentQuestionResultsListItem.jsx

```diff
@@ -11,15 +11,15 @@ import { WysiwygHelper } from '../wysiwyg-helpers'
 import { Responses } from '../api/responses'
 import { _ } from 'underscore'
 
-export class _StudentQuestionResultsListItem extends Component {
+export class StudentQuestionResultsListItem extends Component {
 
   componentWillMount () {
     MathJax.Hub.Queue(['Typeset', MathJax.Hub])
   }
 
   render () {
     const q = this.props.question
-    const unique = _.uniq(q.studentResponses, 'questionId')
+    const unique = _.where(q.studentResponses, { studentUserId: Meteor.userId() })
```

Comment:
Yes that's right, the filtering is unnecessary. I guess I copied the code over from the professor's session review page and didn't change it.